docs/manual/developer_guide.adoc

```diff
@@ -935,3 +939,26 @@ The SSG build and templating system is mostly written in Python.
 * The common pattern is to dynamically add the `shared/modules` to the import path. The `ssgcommon` module has many useful utility functions and predefined constants. See scripts at `./build-scripts` as an example of this practice.
 * Follow the link:https://www.python.org/dev/peps/pep-0008/[PEP8 standard].
 * Try to keep most of your lines length under 80 characters. Although the 99 character limit is within link:https://www.python.org/dev/peps/pep-0008/#maximum-line-length[PEP8 requirements], there is no reason for most lines to be that long.
+
+== Legacy Notice
+
+This project has been created by renaming SCAP Security Guide Project (SSG).
+SSG was originally a project that provides security policies in SCAP format.
+Over the time, the scope of the SSG project has extended over SCAP.
+We started to develop in Bash, Ansible and we introduced a new universal YAML format for
+security content.
+The word SCAP in the project name started to limit our expansion, because people were afraid
+that SCAP is overkill for their needs.
+Therefore, the SCAP Security Guide has been transformed into ComplianceAsCode/content, which better
+described the aim of the project.
+
+This git repository was created by simply renaming and moving the SCAP Security Guide (SSG)
+repository to a different GitHub organization.
+
+However, due to this history, the repository contains many mentions of SCAP Security Guide.
+These legacy mentions are continuously cleaned up as the project develops.
+Some of them are kept due to backwards compatibility.
+
+For example, the output files produced by our build system still start by `ssg-` prefix.
+The reason for this is that the various Linux distributions package ship these files
```

Comment:
Remove "the reason for this is that the".

Also - do we actually plan to keep it like that for backwards compatibility? I believe that's problem of downstream packagers, and we don't have to care too much (when it's one time renaming effort, I mean). File names needs to make sense, that's whole requirement I see.